omotosho-project-c92e4f4bc071.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
don't commit a credential file. Use .env with `decouple` or `python-dotenv` library. Ensure you add your .env to .gitignore